docs/source/developers/ROADMAP_0_4.md

```diff
@@ -0,0 +1,90 @@
+
+- **Improving the performance of operations on in-memory data**. Even when data is loaded in memory, some operations, such as label and shape painting or slicing along large numbers of points can be slow. We will continue developing our [benchmark suite](https://github.com/napari/napari/blob/master/docs/developers/BENCHMARKS.md) and work to integrate it into our development process. See the [`performance` label](https://github.com/napari/napari/labels/performance) for a current list of issues related to performance.
+
+- Add **physical coordinates**. We now have a world coordinate system and transforms that can move between data coordinates, world coordinates, and the canvas where things are rendered; however, we still don't have a concept of phyiscal units. See [#1701](https://github.com/napari/napari/issues/1701) for additional discussion.
+
+- Add **linked multi-canvas support** ([#760](https://github.com/napari/napari/issues/760)) to allow orthogonal views, or linked views of two datasets, for example to select keypoints for image alignment, or simultaneous 2D slices with 3D rendering.
+
+- Add **layer groups** [#970](https://github.com/napari/napari/issues/970), which allow operating on many layers simultaneously making the viewer easier to use for multispectral or multimodal data, or, in the context of multiple canvases, where one wants to assign different groups to different canvases.
+
+- Complete **serialization of the viewer** [#851](https://github.com/napari/napari/pull/851) to enable sharing the entire viewer state, see [#1875](https://github.com/napari/napari/issues/1875) too.
+
+- Support for generating **animations** [#780](https://github.com/napari/napari/pull/780).
+
+- Improved **error handling and bug reporting**, see [#1090](https://github.com/napari/napari/issues/1090) for details.
+
+- Support for persistent settings [#1183](https://github.com/napari/napari/pull/1183) to allow **saving of preferences** between launches of the viewer.
+
+- Improve the **user interface and design** of the viewer to make it easier to use. We have conducted a product heuristics analysis to [identify design and usability issues](https://github.com/napari/product-heuristics-2020), and will now be working during the 0.4 series of releases to implement them. Also see the [`design` label](https://github.com/napari/napari/labels/design) for more information.
+
+## Support **functional and interactive** plugins that allow for image analysis.
+
+- Support for functional plugins that take in layers or layer data and operate on them to return new layers or layer data.
```

Comment:
Already done. Maybe, strengthen support? It would be nice actually to tie it in with the community, e.g. "0.4.3 brought experimental support for functional plugins. During the 0.4.x series, we aim to work with our community to strengthen that support, identify gaps in functionality, and fill those gaps, either with new plugin types or by modifying the existing plugins architecture to suit developer and user needs."

Reply:
yes very open to that. i'd like some of the perspective of the doc to still be from everything in 0.4, but should be up to date with where we are now. can you take this one